keylime/README.md

```diff
@@ -1,4 +1,47 @@
 # Keylime Demo Files
 
 This folder contains the various files needed for Keylime, such as configuration
-autorun scripts and payload / revocation scripts
+autorun scripts and payload / revocation scripts, whitelists and exclude lists.
+
+Using a controller as an example, the following actions would be followed.
```

Comment:
In this case I think we should clarify "Using a controller as an example" Maybe something like using a "VM" as an example, I just think the term "controller" is overused in this space and will lead to confusion.

Reply:
So by controller, I mean the cluster controller..maybe there is a more OpenShift idiomatic name I should be using instead? Any ideas?

Reply:
Yeah, I would go with "node" or "VM" Since a controller is a different openshift/k8's resource that is controlling traffic in the cluster and a bit higher level abstraction. In our case we run keylime on an Openshift Worker node in a container, and then outside the cluster on a VM.  Also, most of this setup would only need to be done for the Keylime on VM instance, since it is already taken care of in the container image

Reply:
For the demo, I think calling it "Keylime Controller" will also do the job.

Reply:
I think we need to go over this on a call.

I perfumed we would have the following:

a, 2 x VM OpenShift nodes
b, 1 x VM OpenShift $XXX machine that will fail over OpenShift Node 1 to Node 2.
c, 1 x VM Keylime Management server running the Verifier and the Registrar

When a node in **a** fails, it migrates the pod over to the second node in **a**

To perform this migration, I assumed we would need a 3rd OpenShift machine **b** to direct the pod to migrate nodes?

I guess we could run **c** on **b**, but it should be separate ideally